src/app-layer-parser.h

```diff
@@ -147,6 +147,12 @@ typedef struct AppLayerGetTxIterState {
     } un;
 } AppLayerGetTxIterState;
 
+typedef struct AppLayerCleanupTxList {
+    uint64_t *tx_id_free_list;
+    // size of tx_id_free_list
+    uint32_t max_tx;
```

Comment:
lets rename this to `tx_id_free_list_size`, then we don't need a comment explaining what it is

Reply:
ok